File: templates/problems/19_acm_io/01_fixed_format/typescript/solution.ts
/**
 * Read n and n integers, return their sum.
 * Time: O(n), Space: O(n) for tokenizing input.
 */
export function solveFixedFormat(inputData: string): string {
    const tokens = inputData.split(/\s+/).filter(t => t.length > 0);
    const n = parseInt(tokens[0], 10);
    let total = 0;
    for (let i = 1; i <= n; i++) {
        total += parseInt(tokens[i], 10);
    }
    return `${total}\n`;
}

File: templates/problems/19_acm_io/01_fixed_format/python/solution.py
def solve_fixed_format(input_data: str) -> str:
    """Read n and n integers, return their sum.

    Time: O(n), Space: O(n) for tokenizing input.
    """
    tokens = input_data.split()
    n = int(tokens[0])
    total = 0
    for i in range(1, n + 1):
        total += int(tokens[i])
    return f"{total}\n"
